app/services/user_sign_up.rb

```diff
@@ -0,0 +1,46 @@
+class UserSignUp
+  attr_reader :error_message
+
+  def initialize(user)
+    @user = user
+  end
+
+  def sign_up(stripe_token, invitation_token)
```

Comment:
I also like keyword (a.k.a named) arguments in cases where there are 2 or more arguments for clarity of signature.
`def sign_up(stripe_token:, invitation_token: nil)`.